Portal/src/Datahub.Portal/Startup.cs

```diff
@@ -218,11 +219,22 @@ public void ConfigureServices(IServiceCollection services)
 
         if (ReverseProxyEnabled())
         {
-            services.AddReverseProxy();
+            services.AddReverseProxy()
+                    .AddTransforms(builderContext =>
+                    {
+                        builderContext.AddRequestTransform(async transformContext =>
+                        {
+                            // passing the logged user to the proxied app
+                            var loggedUser = transformContext.HttpContext?.User?.Identity?.Name ?? "";
+                            transformContext.ProxyRequest.Headers.Add(GetUserHeaderName(), loggedUser);
+                            await Task.CompletedTask;
```

Comment:
Is this simply for the app to know or for an auth piece? What's the behaviour if it's the `""` empty string?